python/utils/PyTypes.h

```diff
@@ -0,0 +1,168 @@
+/****************************************************************-*- C++ -*-****
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#pragma once
+
+#include <complex>
+#include <pybind11/complex.h>
+#include <pybind11/pybind11.h>
+
+#include <iostream>
+
+namespace py = pybind11;
```

Comment:
Should omit namespace aliases and especially `using namespace ship;` from headers.

Reply:
will do, thanks!